tabs/system-setup/fedora/virtualization.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+# Install virtualization tools to enable virtual machines
+configureVirtualization() {
+    case $PACKAGER in
+        dnf)
+             echo "Installing virtualization tools..."
```

Comment:
```suggestion
            echo "Installing virtualization tools..."
```
There's an extra space here, everything needs to be formatted properly.